src/oas3/operation.ts

```diff
@@ -41,6 +42,7 @@ export const transformOas3Operation: Oas3HttpOperationTransformer = ({ document,
       getOasParameters(operation.parameters as ParameterObject[], pathObj.parameters),
       operation.requestBody as RequestBodyObject,
     ),
+    callbacks: translateToCallbacks(operation.callbacks || {}),
```

Comment:
```suggestion
    callbacks: operations.callbacks ? translateToCallbacks(operation.callbacks) : undefined,
```

I know that this code will look different than the other one, but we seriously need to stop this habit that everything can be undefined and somebody else will figure it out.